test/ode_tableau_convergence_tests.jl

```diff
@@ -15,16 +15,26 @@ Random.seed!(100)
 ## Convergence Testing
 println("Convergence Test on Linear")
 dts = 1 .//2 .^(8:-1:4)
-testTol = 0.3
+testTol = 0.7
```

Comment:
that's really high

Reply:
Yes, because for high order methods, it could superconvergence on linear problems.

Reply:
it would be good to not do that on all of them

Reply:
We could check `sim.𝒪est[:l∞] >= tab.order || abs(sim.𝒪est[:l∞] - tab.order) < 0.3` too.